README.md

```diff
@@ -24,30 +24,63 @@ This repository hosts the python library for communicating with the Pocket Scien
 
 To install PSLab on Debian based GNU/Linux system, the following dependencies must be installed.
 
-### Dependencies
-
-* Python 3.4 or higher [Link to Official download page](https://www.python.org/downloads/windows/)
+### Prerequisites & Dependencies
+* Python 3.4 or higher [Link to Official download page](https://www.python.org/downloads/)
 * Pip &nbsp; **Support package installer**
 * NumPy &nbsp; **For numerical calculations**
 * PySerial &nbsp; **For device connection**
 * iPython-qtconsole &nbsp; **_Optional_**
+* [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) 
 
 **Note**: If you are only interested in using PSLab as an acquisition device without a display/GUI, only one repository  [pslab-python](https://github.com/fossasia/pslab-python) needs to be installed. If you like a GUI, install the [pslab-desktop app](https://github.com/fossasia/pslab-desktop) and follow the instructions of the Readme in that repo.
 
 ### How To Install on Linux
 
-cd into the directories
+Clone [pslab-python](https://github.com/fossasia/pslab-python) repository 
+
+    $ git clone <REPO>
+
+Go into **pslab-python** directory
 
     $ cd <SOURCE_DIR>
 
-and run the following
+And run the following
 
     $ sudo make clean
     $ sudo make
     $ sudo make install
 
-Now you are ready with the PSLab software on your machine.
+Now you connect PSLab device to your machine, to check whether device is detected, run `$ ls -l /dev/ttyACMx`: x is 0 for the first device. 
```

Comment:
This is not a good check

Reply:
hello there, I'm new and i am not sure if that was meant for me or not as I only tried to fix grammar and most basic errors...

Reply:
Hello Abhinav, what do you mean by you're not sure?

We use `lsusb` for a connection check.